UnitsNet/CustomCode/Quantities/Force.extra.cs

```diff
@@ -19,59 +19,5 @@ public static Force FromMassByAcceleration(Mass mass, Acceleration acceleration)
         {
             return new Force(mass.Kilograms * acceleration.MetersPerSecondSquared, ForceUnit.Newton);
         }
-
-        /// <summary>Get <see cref="Power"/> from <see cref="Force"/> times <see cref="Speed"/>.</summary>
-        public static Power operator *(Force force, Speed speed)
-        {
-            return Power.FromWatts(force.Newtons * speed.MetersPerSecond);
-        }
-
-        /// <summary>Get <see cref="Power"/> from <see cref="Speed"/> times <see cref="Force"/>.</summary>
-        public static Power operator *(Speed speed, Force force)
-        {
-            return Power.FromWatts(force.Newtons * speed.MetersPerSecond);
-        }
-
-        /// <summary>Get <see cref="ForcePerLength"/> from <see cref="Force"/> times <see cref="ReciprocalLength"/>.</summary>
-        public static ForcePerLength operator *(Force force, ReciprocalLength reciprocalLength)
-        {
-            return ForcePerLength.FromNewtonsPerMeter(force.Newtons * reciprocalLength.InverseMeters);
-        }
-
-        /// <summary>Get <see cref="Pressure"/> from <see cref="Force"/> times <see cref="ReciprocalArea"/>.</summary>
-        public static Pressure operator *(Force force, ReciprocalArea reciprocalArea)
-        {
-            return Pressure.FromNewtonsPerSquareMeter(force.Newtons * reciprocalArea.InverseSquareMeters);
-        }
-
-        /// <summary>Get <see cref="Acceleration"/> from <see cref="Force"/> divided by <see cref="Mass"/>.</summary>
-        public static Acceleration operator /(Force force, Mass mass)
-        {
-            return Acceleration.FromMetersPerSecondSquared(force.Newtons / mass.Kilograms);
-        }
-
-        /// <summary>Get <see cref="Mass"/> from <see cref="Force"/> divided by <see cref="Acceleration"/>.</summary>
-        public static Mass operator /(Force force, Acceleration acceleration)
-        {
-            return Mass.FromKilograms(force.Newtons / acceleration.MetersPerSecondSquared);
-        }
-
-        /// <summary>Get <see cref="Pressure"/> from <see cref="Force"/> divided by <see cref="Area"/>.</summary>
-        public static Pressure operator /(Force force, Area area)
-        {
-            return Pressure.FromPascals(force.Newtons / area.SquareMeters);
-        }
-
-        /// <summary>Get <see cref="ForcePerLength"/> from <see cref="Force"/> divided by <see cref="Length"/>.</summary>
-        public static ForcePerLength operator /(Force force, Length length)
-        {
-            return ForcePerLength.FromNewtonsPerMeter(force.Newtons / length.Meters);
-        }
-
-        /// <summary>Get <see cref="Duration"/> from <see cref="Force"/> divided by <see cref="ForceChangeRate"/>.</summary>
-        public static Duration operator /(Force force, ForceChangeRate forceChangeRate)
-        {
-            return new Duration(force.Newtons / forceChangeRate.NewtonsPerSecond, DurationUnit.Second);
```

Comment:
Does this PR replace the existing operators exactly, or are some new operators also added?

Reply:
Some new multiplication operators are added where they where not defined in a commutative way. For example, `HeatFlux * Area = Power` was defined in `HeatFlux.extra.cs` but the other way round did not exist.
This PR generates both, in `HeatFlux.g.cs` and `Area.g.cs`, based on a single definition in the json file. 

Also, some operators where defined only for `Duration` or `TimeSpan`, some for both. In the definitions file I only use `Duration` as an operand and then generate the `TimeSpan` equivalent too.

This makes the API more consistent, but also has an effect on code size (net +469 lines in e7ba65004) and test coverage. I explicitly chose not to touch any tests for now but to make sure all existing tests pass. Do you have any guidance on this? Should we generate tests for these operators and then remove the manual tests?

Reply:
Ok, those additions sounds good. Keeping the change as small as possible compared to existing code.

Code size is fine I think, I favor consistency.
Yes, let's leave the tests for now to make sure what existed before will still work, then we can generate tests in a second PR after merging this.